### PR TITLE
Fix: Astrocyte grower infinite loop 

### PR DESCRIPTION
**Pull request by jacquemi-bbp**
- append a copy of point into the points list instead of a refence to the point.

### Description
Fix [103](https://github.com/BlueBrain/NeuroTS/issues/103)

### Checklist

This pull request is:

- [ ] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [ ] A short code fix
	- [x] Please include: `Fixes: #<issue number>` in the description if it solves an existing issue
	  (which must include a complete example of the issue).
	- [ ] Please include tests that fail with the `main` branch and pass with the provided fix.
- [ ] A new feature implementation or update an existing feature
	- [ ] Please include: `Fixes: #<issue number>` in the description if it solves an existing issue
	  (which must include a complete example of the feature).
	- [ ] Please include tests that cover every lines of the new/updated feature.
	- [ ] Please update the documentation to describe the new/updated feature.

### CODE REVIEW

File: neurots/astrocyte/section.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def grow_to_target(start_point, start_direction, target_point, segment_length, p
         )
 
         point += segment_length * direction
-        points.append(point)
+        points.append(point.copy())
```
Comment (eleftherioszisis) [inline neurots/astrocyte/section.py:68]: I would prefer if we didn't mutate that point in-place. So instead:
```python
point = point + segment_length * direction
points.append(point)
```
And avoid the in-place mutation altogether.

**Comment by jacquemi-bbp** [comment]
How does the versioning of this package works ?
Do I need to change something for the versioning ?

**Comment by eleftherioszisis** [comment]
> How does the versioning of this package works ? Do I need to change something for the versioning ?

No, I don't think so. Versioning should be automatic when a new release is made.

**Review by arnaudon — APPROVED** [review]

**Comment by arnaudon** [comment]
write `Fix:blabla` in commit title

**Comment by arnaudon** [comment]
ah, this changes the algo a bit apparently, as a test break, I guess you can update the test, it should not be such a big deal, but @eleftherioszisis should know more

**Review by eleftherioszisis — APPROVED** [review]

**Pull request merged**